ATL/schema.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE Passengers(
+passenger_id INT Identity(1, 1) PRIMARY KEY,
+first_name varchar(20) NOT NULL,
+last_name varchar(20) NOT NULL,
+age INT
+);
+
+CREATE TABLE Airlines(
+airline_id INT Identity(1, 1) PRIMARY KEY,
+airline_name varchar(50),
+concourse CHAR(1) CHECK (concourse IN ('A', 'B', 'C', 'D', 'E', 'F', 'T'))
+);

```

Comment:
With this structure we will have many different id's for the same Airline. How are we going to reference the Airline Delta from other tables if we can't uniquely identify it? Here you should have implemented a Many-to-Many relationship from Airline to a Concourse(different table).